onnxruntime/core/providers/qnn/builder/qnn_backend_manager.h

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 #pragma once
+#include <utility>
```

Comment:
Is this required?

Reply:
For std::move. Github action linter complains about it.

Reply:
Removed after change to constructor args (std::string&& backend_path)